packages/embed/src/utils/create-iframe/setup-custom-keyboard-close.ts

```diff
@@ -0,0 +1,25 @@
+const POST_MESSAGE = 'form-close'
+const KEY_EVENT = 'Escape'
+
+const closeForm = async (close: () => void) =>
```

Comment:
why do we need this?

Reply:
I wait for the form to be close before removing the `eventListener`